db/db.go

```diff
@@ -285,10 +285,26 @@ func (d *DB) Reduce(events []core.Event) error {
 	return nil
 }
 
+// GetInviteInfo returns the addresses and key that can be used to join the DB thread
+func (d *DB) GetInviteInfo() (thread.ID, []ma.Multiaddr, thread.Key, error) {
+	ctx := context.Background()
+	creds := d.adapter.threadCreds
+	addresses, err := d.adapter.net.GetThreadAddresses(ctx, creds)
+	if err != nil {
+		return thread.ID{}, nil, thread.Key{}, err
+	}
+	tinfo, err := d.adapter.net.GetThread(ctx, creds)
+	if err != nil {
+		return thread.ID{}, nil, thread.Key{}, err
+	}
+	return d.adapter.threadCreds.ThreadID(), addresses, tinfo.Key, nil
+}
+
```

Comment:
Then the `DB` can use the two methods from `Net` to build the information needed to invite another peer to a thread.

Reply:
Little update since above comment... This now just calls the underlying `net.GetThread` and returns the useful parts of the result.

Reply:
This is were DB needs access to the net connector, and some other places below as well.